lib/flux/app-state.js

```diff
@@ -397,49 +396,15 @@ export const actionMap = new ActionMap({
       creator({ noteBucket, noteId, data, remoteUpdateInfo = {} }) {
         return (dispatch, getState) => {
           var state = getState().appState;
-          const { original, patch } = remoteUpdateInfo;
+          const { patch } = remoteUpdateInfo;
```

Comment:
The `patch` should still be used to maintain the cursor location between updates.